templates/zerver/integrations/hubot_common.md

```diff
@@ -1,8 +1,14 @@
+# Zulip {{ integration_display_name }} integration
+
 Get Zulip notifications from {{ integration_display_name }} via Hubot!
 
+{start_tabs}
+
 1.  {!create-channel.md!}
 
```

Comment:
I think we can just cut the "create a channel" instruction. It doesn't even make sense given how Hubot is described to work.